app/src/main/java/com/chesire/nekome/Activity.kt

```diff
@@ -110,6 +115,16 @@ class Activity : DaggerAppCompatActivity(), AuthCaster.AuthCasterListener, Flow
                 }
             }
         }
+
+        if (settings.defaultHomeScreen == HomeScreenOptions.Anime) {
```

Comment:
This extra statement at the end flags up a Detekt warning for the method being too long, maybe this section can be extracted out into its own method that `finishLogin` also uses?